### PR TITLE
Show users with same mail address and name

### PR DESCRIPTION
**Pull request by KatrinIhler**
Always use the username as key when sending users lists to the admin ui since only that is supposed to be unique. If it's the other way around, users with the same name and mail address cannot be added to a group that a similar user is already in.

### CODE REVIEW

**Comment by lkiesow** [comment]
@lwillmann, can you review this patch?

File: etc/org.opencastproject.organization-mh_default_org.cfg [hunk 1]
```diff
@@ -168,11 +168,11 @@ prop.adminui.user.external_role_display=false
...
-# - USERS.INVERSE will list users by name (e.g. "John Sample")
-# - USERS.INVERSE.WITH.EMAIL will list users by name and email (e.g. "John Sample <j.sample@opencast.org>")
-# - USERS.INVERSE.WITH.USERNAME will list users by name and username (e.g. "John Sample (jsample)")
-# Default: USERS.INVERSE.WITH.USERNAME
-#prop.adminui.user.listname=USERS.INVERSE.WITH.USERNAME
+# - USERS.NAME will list users by name (e.g. "John Sample")
+# - USERS.NAME.AND.EMAIL will list users by name and email (e.g. "John Sample <j.sample@opencast.org>")
+# - USERS.NAME.AND.USERNAME will list users by name and username (e.g. "John Sample (jsample)")
+# Default: USERS.NAME.AND.USERNAME
+prop.adminui.user.listname=USERS.NAME.AND.EMAIL
```
Comment (lwillmann) [inline etc/org.opencastproject.organization-mh_default_org.cfg]: Please don't change the default configuration

**Review by lwillmann — CHANGES_REQUESTED** [review]
Looks good for me, except the mentioned change in the configuration.

File: etc/org.opencastproject.organization-mh_default_org.cfg (hunk 1 above)
Comment (KatrinIhler) [reply etc/org.opencastproject.organization-mh_default_org.cfg]: Whoops, this wasn't intended. Fixed!

**Review by lwillmann — APPROVED** [review]

**Pull request merged**